legal-api/src/legal_api/services/filings/validations/common_validations.py

```diff
@@ -18,6 +18,18 @@
 from legal_api.utils.datetime import datetime as dt
 
 
+def has_at_least_one_share_class(filing_json, filing_type) -> Error:  # pylint: disable=too-many-branches
```

Comment:
This should get wrapped by the type Optional,  and maybe be a str?

Reply:
fix